### DISCUSSION TITLE
Nerf zombie scent

### DISCUSSION THREAD

**Discussion by JollyR0ger42** [CleverRaven/Cataclysm-DDA#45981] [Ideas]
I think zombie scent should be nerfed, Now u dont have abilities to crouch between zombies in dark room.
IRL people dont have such precision (at least i think so).
Sugestion:
 - nerf scent for regular zombie
 - boost scent for zombie dogs and give them abilities to bark(not sure if they have it now)

UPD: make normal zombie react to blood scent, if player have some type of bleeding

**Comment by Rivet-the-Zombie** [comment]
Reducing the effectiveness of most 'normal ' zombies' noses would also set us up for the possibility of having various special 'sniffer' variants that could track you down like a bloodhound.

**Reply by JollyR0ger42** [reply]
Doesn't we have them now? Cuz zombie listeners in game, make sense to have sniffers as well

**Comment by kevingranade** [comment]
Need a bit more clarity, nerf to what? In what situations is it too good?
Are you saying remove it instead?

**Reply by JollyR0ger42** [reply]
i think much slower reaction will do the trick, for example time threshold for 60 seconds for zombie to smell player after step in his scent cloud

**Reply by JollyR0ger42** [reply]
Good for more stealthy gameplay

**Comment by Entity-y** [comment]
Instead of nerfing the zombie's scent chasing ability, have certain factors reduce the scent trail. Smoking could mostly get rid of that human smell, as well as wearing filthy clothes or covering yourself in tainted blood - at the cost of a massive mood penalty.

**Reply by robotic-gamer** [reply]
some other ideas
-washing yourself would reduce your scent, especially if you use soap
-a stink bomb could act as a distraction (could be crafted from sulfur, ammonia and lamp oil)
